src/examples/databases/src/frontend/types.generated.ts

```diff
@@ -385,14 +387,10 @@ export type TasksListFilter = {
   dueAt_nin?: InputMaybe<Array<Scalars['Date']['input']>>;
   dueAt_notnull?: InputMaybe<Scalars['Boolean']['input']>;
   dueAt_null?: InputMaybe<Scalars['Boolean']['input']>;
-  id?: InputMaybe<Scalars['ID']['input']>;
-  id_gt?: InputMaybe<Scalars['ID']['input']>;
-  id_gte?: InputMaybe<Scalars['ID']['input']>;
-  id_in?: InputMaybe<Array<Scalars['ID']['input']>>;
-  id_lt?: InputMaybe<Scalars['ID']['input']>;
-  id_lte?: InputMaybe<Scalars['ID']['input']>;
-  id_ne?: InputMaybe<Scalars['ID']['input']>;
-  id_nin?: InputMaybe<Array<Scalars['ID']['input']>>;
+  id?: InputMaybe<Scalars['BigInt']['input']>;
+  id_in?: InputMaybe<Array<Scalars['BigInt']['input']>>;
+  id_ne?: InputMaybe<Scalars['BigInt']['input']>;
+  id_nin?: InputMaybe<Array<Scalars['BigInt']['input']>>;
```

Comment:
I reckon we should still be generating the comparison operators for `BigInt` scalars. Maybe that's also special handling in the Schema Generator?

Reply:
Decided this is a different task, all good!